tests/speculos/test_configuration_cmd.py

```diff
@@ -0,0 +1,10 @@
+
+def test_configuration(cmd):
+    if cmd.model == "nanos":
+        assert cmd.get_configuration() == (14, 1, 9, 20)
```

Comment:
What are these numbers? Could it be explicited?

Is there a meaning to check the model, if all results are the same?

Reply:
I was thinking of letting it separate if at some point the versions change with respect to the device (s,x,sp)

Reply:
Alright